spec/attr_extras/attr_initialize/attributes_declaration_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "spec_helper"
+
+describe AttrExtras::AttrInitialize::AttributesDeclaration do
```

Comment:
This test is great – makes it a lot easier to understand what the code does, and to trust it.